crates/core/src/test_utils/mod.rs

```diff
@@ -1,4 +1,5 @@
 mod factories;
+pub mod kms_encryption;
```

Comment:
We should add `#[cfg(feature = "datafusion")]` here so we don't need it on everything in `kms_encryption`

Reply:
Sounds good.